bchrpc/server.go

```diff
@@ -844,7 +884,8 @@ func (s *GrpcServer) GetRawAddressTransactions(ctx context.Context, req *pb.GetR
 	// Attempt to decode the supplied address.
 	addr, err := bchutil.DecodeAddress(req.Address, s.chainParams)
 	if err != nil {
-		return nil, status.Error(codes.InvalidArgument, "invalid address")
+		addr, err = goslp.DecodeAddress(req.Address, s.chainParams)
```

Comment:
Yeah we should have a high level function that takes care of this flow.

Reply:
done.